README.md

```diff
@@ -8,435 +8,429 @@ ACL (Access Control List)
-      permissions => [
-       { identity => 'SYSTEM', rights => ['modify'], child_types => 'none' },
-       { identity => 'SYSTEM', rights => ['modify'], child_types => 'containers' },
-       { identity => 'SYSTEM', rights => ['modify'], child_types => 'objects' },
-       { identity => 'SYSTEM', rights => ['full'], affects => 'self_only' },
-       { identity => 'SYSTEM', rights => ['read','execute'], affects => 'direct_children_only' },
-       { identity => 'SYSTEM', rights => ['read','execute'], child_types=>'containers', affects => 'direct_children_only' },
-       { identity => 'SYSTEM', rights => ['read','execute'], child_types=>'objects', affects => 'direct_children_only' },
-       { identity => 'SYSTEM', rights => ['full'], affects => 'children_only' },
-       { identity => 'SYSTEM', rights => ['full'], child_types=>'containers', affects => 'children_only' },
-       { identity => 'SYSTEM', rights => ['full'], child_types=>'objects', affects => 'children_only' },
-       { identity => 'SYSTEM', rights => ['read'], affects => 'self_and_direct_children_only' },
-       { identity => 'SYSTEM', rights => ['read'], child_types=>'containers', affects => 'self_and_direct_children_only' },
-       { identity => 'SYSTEM', rights => ['read'], child_types=>'objects', affects => 'self_and_direct_children_only' }
-      ],
-      inherit_parent_permissions => 'false',
-    }
-```
+With Windows, you can specify the same `identity` with different inheritance and propagation. Each of the resulting items is managed as a separate ACE.
+
+~~~
+acl { 'c:/tempperms':
+  purge                      => 'true',
+  permissions                => [
+   { identity => 'SYSTEM', rights => ['modify'], child_types => 'none' },
+   { identity => 'SYSTEM', rights => ['modify'], child_types => 'containers' },
+   { identity => 'SYSTEM', rights => ['modify'], child_types => 'objects' },
+   { identity => 'SYSTEM', rights => ['full'], affects => 'self_only' },
+   { identity => 'SYSTEM', rights => ['read','execute'], affects => 'direct_children_only' },
+   { identity => 'SYSTEM', rights => ['read','execute'], child_types=>'containers', affects => 'direct_children_only' },
+   { identity => 'SYSTEM', rights => ['read','execute'], child_types=>'objects', affects => 'direct_children_only' },
+   { identity => 'SYSTEM', rights => ['full'], affects => 'children_only' },
+   { identity => 'SYSTEM', rights => ['full'], child_types=>'containers', affects => 'children_only' },
+   { identity => 'SYSTEM', rights => ['full'], child_types=>'objects', affects => 'children_only' },
+   { identity => 'SYSTEM', rights => ['read'], affects => 'self_and_direct_children_only' },
+   { identity => 'SYSTEM', rights => ['read'], child_types=>'containers', affects => 'self_and_direct_children_only' },
+   { identity => 'SYSTEM', rights => ['read'], child_types=>'objects', affects => 'self_and_direct_children_only' }
+  ],
+  inherit_parent_permissions => 'false',
+}
+~~~
+
+
+##Reference
```

Comment:
Interesting why reference moved to the bottom. Most folks who are not learning would be here more often. I guess that becomes a matter of preference. I'd almost go  basics, then reference, then more detailed examples.